physionet-django/project/models.py

```diff
@@ -756,7 +841,8 @@ def get_anonymous_url(self):
 
         return anonymous.url
 
-    def citation_text(self, style):
+    def citation_text(self, style, platform='Physionet', authors=None,
```

Comment:
"Physionet" should be "PhysioNet", but I'm not sure we want this as a default argument (we want to avoid hardcoding this word where possible).

Reply:
Yeah, I knew this would come up, has your global variable published yet? Oh wait, that would cause an error since I'm behind that push anyway

Reply:
I set it as a default since the platform would be used most often but I can remove it if that's better?